packages/@o3r/core/schematics/module/templates/base/README.md.template

```diff
@@ -0,0 +1,17 @@
+# <%= name %>
+
+This package is an [Otter Framework Module](https://github.com/AmadeusITGroup/otter/tree/main/docs/core/MODULE.md).
+
+<!-- TODO: include module summarize -->
+
+## How to install
+
+```shell
+ng add <%= name %>
+```
+
+> **Warning**: this module requires [@o3r/core](https://www.npmjs.com/package/@o3r/core) to be installed.
+
+## Description
+
```

Comment:
maybe we can at least include `<%= description %>`

Reply:
I included it in `summarize` section